icons/book-tabs.svg

```diff
@@ -0,0 +1,16 @@
+<svg
```

Comment:
I doubt this one as well. If it is about contact/address book is `contact` or `contact-2` not a better icon for this?

Reply:
> is about contact/address book is contact or contact-2 not a better icon for this?

I would say those represent a single `contact` vs multiple (a list/book/app full of them…)

Reply:
Hmm ok, If we want use this for contact lists, maybe a design something like this would be more clear:
![image](https://github.com/lucide-icons/lucide/assets/11825403/8c5d1538-f007-4231-994c-84f0205db12b)